### PR TITLE
Feature/hero section

### PR DESCRIPTION
**Pull request by JamesDrysdale**
Hero section with working contact me button which copies my email to the user's clipboard. The my work button doesn't go anywhere yet, but I will add that in when I create the section it will be navigating to.

Previews:

Screenshot 2023-09-11 at 02 58 24
Screenshot 2023-09-11 at 02 58 41
Screenshot 2023-09-11 at 02 59 11
Screenshot 2023-09-11 at 02 59 26

### CODE REVIEW

File: components/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            My Work
+          </a>
+          <CopyEmail email={'jamesdrysdale84@gmail.com'} />
+        </div>
+      </div>
+      {/* Image */}
+      <div className='self-center'>
+        <Image
+          src={heroImg}
+          alt='hero image icon'
```
Comment (cdsalty) [inline components/Hero.tsx:40]: Try to remember to use meaningful 'alt' tags such as ` alt='Portrait of James, a Web Developer'`

File: components/ThemedImage.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import { useTheme } from 'next-themes';
+
+function ThemedImage(fileName: string | any) {
```
Comment (cdsalty) [inline components/ThemedImage.tsx:4]: Did you mean to include 'any' here? Using 'any' as a type in TypeScript, effectively disables type checking for this part of the code, negating the main benefits (type-checking) of using TypeScript.

**Review by cdsalty — APPROVED** [review]
Hi @JamesDrysdale! This looks great! I made a couple suggestions to implement on your next Pull Request. Keep up the good work.

Cheers!

File: components/Hero.tsx (hunk 1 above)
Comment (JamesDrysdale) [reply components/Hero.tsx:40]: Thank you. I thought I had updated that - well spotted. :)

File: components/ThemedImage.tsx (hunk 2 above)
Comment (JamesDrysdale) [reply components/ThemedImage.tsx:4]: You're absolutely right. I'll take another look at this. Thanks :D

**Pull request merged**